pkg/invoke/delegate.go

```diff
@@ -33,7 +34,7 @@ func delegateAddOrGet(command, delegatePlugin string, netconf []byte, exec Exec)
 		return nil, err
 	}
 
-	return ExecPluginWithResult(pluginPath, netconf, ArgsFromEnv(), exec)
+	return ExecPluginWithResult(context.TODO(), pluginPath, netconf, ArgsFromEnv(), exec)
```

Comment:
Do the same for these, have callers pass their own Context to DelegateAdd, DelegateGet, and DelegateDel()

Reply:
Ok,i will change this api interface~

Reply:
@liucimin any chance you can update the PR for this change?